src/dummy_bus.rs

```diff
@@ -0,0 +1,86 @@
+#![allow(unused_variables)]
+
+use crate::bus::UsbBus;
+
+/// Dummy bus implementation with no functionality.
+///
+/// Examples can create an instance of this bus just to make them compile.
+pub struct DummyUsbBus;
```

Comment:
As eldruin requested, mind renaming this module to `dummy.rs`? Outside of that, I'm fine merging this and seeing how it goes without incorporating a feature.

Reply:
> As eldruin requested, mind renaming this module to `dummy.rs`? Outside of that, I'm fine merging this and seeing how it goes without incorporating a feature.

Done.